e2e/testconfig/testconfig.go

```diff
@@ -69,7 +71,7 @@ func FromEnv() TestConfig {
 
 	chainATag, ok := os.LookupEnv(ChainATagEnv)
 	if !ok {
-		panic(fmt.Sprintf("must specify %s version for test with environment variable [%s]", chainBinary, ChainATagEnv))
+		chainATag = defaultChainTag
```

Comment:
this lets us run the tests without setting any env vars. The default config will use `simd` images tagged with `main`. And the go relayer at `main` as well.